core/resources/polygon.fs

```diff
@@ -1,12 +1,19 @@
-#ifdef GL_ES
-precision mediump float;
-#endif
+uniform mat4 u_modelView;
+uniform mat4 u_modelViewProj;
+uniform float u_time;
+
+#pragma tangram: material
+#pragma tangram: frag_lighting
```

Comment:
I'd prefer to have these be slightly more verbose but clearer: `fragment_lighting`, `vertex_lighting`.